copter/source/docs/autotrim.rst

```diff
@@ -25,54 +25,49 @@ Save trim is the simpler method and involves essentially transferring your radio
+5. Arm the vehicle and lift into a hover. Flip the switch that you setup to  its high position (>1800us).
 
-6. Reset your transmitters roll and pitch trims back to the center and fly again and it should fly level now. If it does not repeat steps 3, 4 & 5
+6. Fly your copter for about 25 seconds in a stable hover in position, keeping it from drifting with the pitch and roll sticks. As the trim is adjusted, less and less corrections with the sticks will be required.
 
-Auto Trim
-~~~~~~~~~
+7. Once satisfied, flip the switch low, saving the trims, and land.
 
-With auto trim the roll and pitch trim are captured as you fly in a stable hover.
+8. Take off again in stabilize mode and check if your copter is flying level now. If not repeat steps 4 - 7 again.
 
-1. Find a wind free environment with sufficient space to fly your copter without crashing into something
-
-2. Put the vehicle in Stabilize mode
-
-3. Hold throttle down and rudder right for 15seconds or until you see the small red, blue and yellow leds flash in a cyclic pattern
+.. note::
 
-4. Fly your copter for about 25 seconds in a stable hover
+     You can manually set the trim by modifying the :ref:`AHRS_TRIM_X <AHRS_TRIM_X>` and :ref:`AHRS_TRIM_Y <AHRS_TRIM_Y>`. Roll trim is :ref:`AHRS_TRIM_X <AHRS_TRIM_X>`, Pitch trim is :ref:`AHRS_TRIM_Y <AHRS_TRIM_Y>`. Both values are in radians with left roll and forward pitch being negative numbers.
 
-5. Land and put your throttle to zero and wait for a few seconds (the trims parameters are being saved)
+.. note::
 
-6. Take off again in stabilize mode and check if your copter is flying level now. If not repeat steps 2, 3 and 4
+    It is nearly impossible to get rid of all drift so that your copter remains completely motionless without any input.
 
-.. note::
+Save Trim
+~~~~~~~~~
+.. note:: the above AUTO TRIM method is preferred since it does not involve moving transmitter trims after RC calibration.
 
-    You can also test these procedures above are operational on the ground with your battery disconnected. Connect your autopilot to the mission planner and observing the Flight Data screen as you simulate completing the steps above.
+Save trim involves essentially transferring your radio transmitter's trims to correct the autopilot's accelerometer trim. Normally, once :ref:`common-accelerometer-calibration` is done, you never want to change the transmitter's trims, but in this case it is done **temporarily**. The "Save Trim" function ("5") is assigned to any free RC channel, usually a switch based one, using an :ref:`common-auxiliary-functions`. For the example below, Channel 7 will be used.
 
-.. image:: ../images/MP_SaveTrim_FlightDataScreen.jpg
-    :target: ../_images/MP_SaveTrim_FlightDataScreen.jpg
+1. Check that your CH7 switch goes above 1800 on the MissionPlanner's Hardware > Mandatory Hardware > Radio Calibration screen
```

Comment:
It may not be "CH7" I guess

Reply:
It's specifically says that we're using channel 7 as the example for the following

Reply:
OK then, merge away!

Reply:
but needed to add it in the other section....done